crates/runner-host/crates/web-server/public/board.js

```diff
@@ -345,6 +345,9 @@ class Board {
       this._components.push(component);
     }
     this._drawer.appendStatus("Board is ready!");
+    this._channel.send({
+      type: "board_ready",
```

Comment:
I agree it's a bit ugly, but right now the messages from runner to board use `type` while those from board to runner use `componentType`. Do you see a better alternative?

```suggestion
      componentType: "board_ready",
```